packages/ui/lib/components/Footer/EmailSubscriptionForm.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import { Button } from '@unlock-protocol/ui'
+
+export const EMAIL_SUBSCRIPTION_FORM = {
+  portalId: '19942922',
+  formGuid: '868101be-ae3e-422e-bc86-356c96939187',
+}
```

Comment:
Should not be in UI library. This should be passed as props.

Reply:
Yes but these are re-usable components. The package is published to npm. We should not add these constants which we use directly.

Reply:
I agree with @searchableguy here.